### PR TITLE
draft sadness: instanceof/typeof and wrapper types

### PR DESCRIPTION
**Pull request by johannesvollmer**
As a continuation of #14, this time as a draft

### CODE REVIEW

File: _categories/strings.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 ---
-title: "Inappropriate String"
-description: "Some things are strings which shouldn't be strings"
-why-bad: "Results in unnecessary and potentially buggy parsing and stringifying code."
+title: "String Types"
+description: "Some values are stored as strings but shouldn't be"
+why-bad: "Results in unnecessary, error prone parsing and stringifying code."
```
Comment (Novalis15) [inline _categories/strings.md]: I might have gotten something wrong here, please verify my changes :)

File: _sadnesses/instanceof-typeof.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+For example, `typeof null === 'object'` but `null instanceof Object === 'false'`.
+This is because `Object` is a function, and not a primitive type.
+The value `null` does not have a prototype.
+  
+Furthermore, `typeof 'text' === 'string'` but `'text' instanceof String === 'false'`,
+and `typeof 0 === 'number'` but `0 instanceof Number === 'false'`.
+These values also do not have a prototype.
+
+__The catch: `typeof ({x:1}) === 'object'` and `({x:1}) instanceof Object === 'true'`.__
```
Comment (Novalis15) [inline _sadnesses/instanceof-typeof.md]: was __Now the catch:__ before. Changed to __The catch:__ so it sounds a bit more "scientific".
I also set all the "__do__" statements to "do".

File: _sadnesses/primitive-wrappers.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+should not be confused with the 
+[fundamental types](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects),
+which are built-in classes like `Object`, `String` or `Number`.
+These prototypes are wrapper types which can be constructed 
+using a _real_ number or a _real_ string.
+
+Even though strings don't inherit from `String`,
+JavaScript automatically wraps strings into a `String` where required,
+such that all of `String`s methods can be called on a string.
+Sounds like inheritance at first, but is not.
```
Comment (Novalis15) [inline _sadnesses/primitive-wrappers.md]: was `Sounds like inheritance to me, but is not.` Sounds very subjective, which is why i changed it to `Sounds like inheritance at first, but is not`. Also considering to write `..., but it's not` or so.

**Comment by johannesvollmer** [comment]
Sounds good

**Pull request merged**